examples/with-nestjs/src/queues/basic-auth.middleware.ts

```diff
@@ -30,7 +31,10 @@ export class BasicAuthMiddleware implements NestMiddleware {
       return;
     }
 
-    const isPasswordValid = await await bcrypt.compare(password, this.passwordHash);
+    const isPasswordValid = await await bcrypt.compare(
```

Comment:
why there is 2 `await` here?

Reply:
maybe I forgot to save before commit. will push a fix